core/src/main/resources/defaults/blueprints/cdp-sdx-medium-ha-722.bp

```diff
@@ -20,7 +20,11 @@
           "kafka-GATEWAY-BASE",
           "kafka-KAFKA_BROKER-BASE",
           "solr-SOLR_SERVER-BASE",
-          "ranger-RANGER_ADMIN-BASE"
+          "ranger-RANGER_ADMIN-BASE",
+          "hdfs-GATEWAY-BASE",
+          "kafka-GATEWAY-BASE",
+          "hive-GATEWAY-BASE",
+          "hbase-GATEWAY-BASE"
```

Comment:
Why no solr-GATEWAY-BASE in the master node?

Reply:
missed it. good catch, https://github.com/hortonworks/cloudbreak/pull/8904